Updated to include list page and new screenshots (#32881)
DataDog/documentation: content/en/profiler/automated_analysis.md

Click an insight to see a high-level summary that explains the issue, contextual insights from profiling data, and recommended next steps.
{{< img src="profiler/profiling_automated_analysis_detail.png" alt="Expanded Profiling Insights showing the details of a detected Issue" style="width:100%;">}}

The Insights list page provides a centralized view of all detected issues across your services. It helps teams quickly understand what’s happening in their environments, prioritize what to investigate, and track whether recurring problems are improving over time.

{{< img  src="profiler/profiling_automated_analysis_list_page_home.png" alt="The list page showing detected insights across services" style="width:100%;">}}

Each row represents an insight type, summarizing:

- Service and runtime affected
- Insight type (for example, GC Pauses or High Lock Contention)
- Severity (for example, Info or Warning)

> You can filter insights by runtime, service, o environment to narrow the list to what matters most. Teams often use this view to identify patterns—such as multiple services affected by the same inefficiency. Clicking on an insight opens its detail panel.

Reviewer: ```suggestion
You can filter insights by runtime, service, or environment to narrow the list to the most important insights. Teams often use this view to identify patterns, such as multiple services affected by the same inefficiency. Clicking on an insight opens its detail panel.
```
